src/views/Rules/ListRules/index.jsx

```diff
@@ -416,6 +417,35 @@ function ListRules(props) {
           // diff viewer
           height += diffedProperties.length * 21 + theme.spacing(5);
         }
+
+        if (Object.keys(rule.scheduledChange.required_signoffs).length > 0) {
+          const requiredRoles = Object.keys(
+            rule.scheduledChange.required_signoffs
+          ).length;
+          const nSignoffs = Object.keys(rule.scheduledChange.signoffs).length;
+
+          // Add space for the padding above the summary,
+          // and at the bottom of the section
+          height += theme.spacing(2);
+
+          // Add space for the "Requires Signoff From" title
+          height += theme.spacing(3);
+
+          // Add space for each row of required roles
+          // 2 is actually a little less than needed, but we rarely
+          // require more than 2 signoffs, and erring low looks better
+          // than erring high.
+          height += requiredRoles * theme.spacing(2);
```

Comment:
> I think we could do Math.max(nSignoffs, requiredRoles) and remove the extra logic below.

Good point, will do.

> Also, on my end the row height is a bit over 20px which means theme.spacing(2) may be not be enough. Are you seeing a different row height on your end?

Yup, I've got ~18.5px here.

Reply:
Interesting! Ok, I will try to investigate to see if there's anything we can do about it. Worst case we just support the highest height (i.e., `theme.spacing(3)` instead of `theme.spacing(2)`).

Reply:
Yeah, we can do that. It's slightly worse looking on my screen, but not dealbreaking.